tasks/Debian-tarball.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: tarball.yml
+
+- include: upstart.yml
+  tags: deploy
+
+- include: systemd.yml
+  tags: deploy
```

Comment:
> Checked 15.04 ...15.10
> it contains both init and systemd which might lead to unexpected behavior
After some reading I'm quite confident that there shouldn't be a problem to write the configs for both init systems (as only one should be active at the same time).
Ref: https://wiki.ubuntu.com/SystemdForUpstartUsers

Anyway I will now push the proposed+agreed solution, meaning that any user that deviated from the default init system (for ex. when on Ubuntu 15.04 he switched to upstart) has to overwrite the default.